Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -70,6 +70,26 @@ extension Option: DecodableParsedWrapper where Value: Decodable {}
 // MARK: Property Wrapper Initializers
 
 extension Option where Value: ExpressibleByArgument {
+  /// Creates a property with an optional default value, intended to be called by other constructors to centralize logic.
+  ///
+  /// This private `init` allows us to expose multiple other similar constructors to allow for standard default property initialization while reducing code duplication.
+  private init(
+    name: NameSpecification = .long,
+    initial: Value? = nil,
+    parsingStrategy: SingleValueParsingStrategy = .next,
+    help: ArgumentHelp? = nil
+  ) {
+    self.init(_parsedValue: .init { key in
+      ArgumentSet(
+        key: key,
+        kind: .name(key: key, specification: name),
+        parsingStrategy: ArgumentDefinition.ParsingStrategy(parsingStrategy),
+        parseType: Value.self,
+        name: name,
+        default: initial, help: help)
+      })
+  }
```

Comment:
Nice job pulling this out 👍

Reply:
Thanks! And not sure what happened in the `transform` equivalent of this; I had that mirroring this one but I guess didn't actually stage/commit all of it over the weekend so it was in a broken state here 😅 

That's been fixed and pushed now though.